lzy/graph-executor/src/test/resources/log4j2.yaml

```diff
@@ -14,14 +14,5 @@ Configuration:
       AppenderRef:
         ref: Console
     Logger:
-      - name: UserEventLogs
```

Comment:
don't remove these settings, plz :)

just change `ai.lzy.graph` from `debubg` to `info`